community/kernel/src/main/java/org/neo4j/kernel/impl/util/Cursors.java

```diff
@@ -45,13 +43,31 @@ public boolean next()
         @Override
         public Object get()
         {
-            return null;
+            throw new IllegalStateException( "no elements" );
```

Comment:
Is this in line with Cursor contract? If we change it we should definitely change (or in this case _add_) javadoc to Cursor mentioning this.